### DISCUSSION TITLE
why doesnt this work

### DISCUSSION THREAD

**Discussion by CashierNub** [FabricMC/fabric-api#2263] [Player support]
i had this error called
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'fabric-api-lookup-api-v1'!
can someone fix it?????

**Comment by strikeknight57** [comment]
I think I might have a similar problem, what processor are you running and what are the specs of the rest of your build?

**Reply by Linguardium** [reply]
this issue appears to be related to a mod that is being used for the wrong version of minecraft. please be sure to paste the full log and give us the link.
http://mclo.gs/ will hide your personal info if there is any
